data/qc_config.yml

```diff
@@ -7,11 +7,6 @@ sea_water_temperature: # deg_C
     sensor_span:
       - -5.
       - 45.
-  rate_of_change:
-    thresh_val: 5.
-  spike:
-    low_thresh: 5.
-    high_thresh: 10.
```

Comment:
I can revert that commit if you like. Those values are being calculated now, not static values.